templates/course_edit.html

```diff
@@ -178,9 +178,9 @@ <h3>
               <dd>
                 <div class="form-group">
                   <div class="fit_wysiwyg">
-                    <div id="fit_wysiwyg_editor">{% if course.summary_html %}{{ course.summary_html|safe }}{% endif %}</div>
+                    <div id="fit_wysiwyg_editor" data-fit-wysiwyg-preview>{% if course.summary_html %}{{ course.summary_html|safe }}{% endif %}</div>
                   </div>
-                  <textarea id="course_summary" name="course_summary" style="display: none"></textarea>
+                  <textarea id="course_summary" name="course_summary" style="display: none" data-fit-wysiwyg-textarea></textarea>
```

Comment:
If there are no other text areas within `data-fit-wysiwyg` divs, you can just search for `p.querySelector('textarea')` and not need this data attribute.

Reply:
Indeed, that's no point in duplicating stuff.